broker_app.py

```diff
@@ -34,166 +35,164 @@
                     format=format)
 
 
-def create_app():
-    app = Flask(__name__, static_folder='static')
-    app.SPREADSHEET_STORAGE_DIR = os.environ.get('SPREADSHEET_STORAGE_DIR')
-    app.SPREADSHEET_UPLOAD_MESSAGE = "We’ve got your spreadsheet, and we’re currently importing and validating the data. \
-Nothing else for you to do - check back later."
-
-    app.SPREADSHEET_UPLOAD_MESSAGE_ERROR = "We experienced a problem while uploading your spreadsheet"
-    app.secret_key = 'cells'
-    cors = CORS(app, expose_headers=["Content-Disposition"])
-    app.config['CORS_HEADERS'] = 'Content-Type'
-
-    app.ingest_api = IngestApi()
-    spreadsheet_generator = SpreadsheetGenerator(app.ingest_api)
-    app.spreadsheet_job_manager = SpreadsheetJobManager(spreadsheet_generator, app.SPREADSHEET_STORAGE_DIR)
-
-    app.register_blueprint(upload_bp)
-    app.register_blueprint(submissions_bp)
-    app.register_blueprint(import_geo_bp)
-    return app
-
-
-app = create_app()
-
-
-@app.route('/', methods=['GET'])
-def index():
-    new_ui_url = os.environ.get('INGEST_UI')
-    if new_ui_url:
-        return redirect(new_ui_url, code=302)
-    return app.response_class(
-        response=json.dumps({'message': "Ingest Broker API is running!"}),
-        status=200,
-        mimetype='application/json'
-    )
-
-
-@app.route('/projects/<project_uuid>/summary', methods=['GET'])
-def project_summary(project_uuid):
-    project = app.ingest_api.get_project_by_uuid(project_uuid)
-    summary = SummaryService().summary_for_project(project)
+def define_app(app):
```

Comment:
Well done for extracting into a function.
I would name this function `add_routes`

Reply:
Applied.